scripts/enforce-settings-gradle-match-directories

```diff
@@ -6,7 +6,7 @@ self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
 diff -u \
-  <(git ls-files --recurse-submodules */ | uniq | grep -E 'build.gradle'  | awk '$1=$1' FS="/" OFS=":" | awk '{gsub("build.gradle", "");print}' | sed 's/.$//' |  sed  's/gitCore:jGit/&\nqual/g') \
+  <(git ls-files '**/build.gradle' | uniq | tr / : | sed 's/:build.gradle//') \
```

Comment:
Nit: `uniq` seems redundant, full file **paths** within the repo can't possibly be duplicated (file **names** could be, but not full paths)